src/job.rs

```diff
@@ -1215,6 +1216,22 @@ pub async fn add(context: &Context, job: Job) {
     }
 }
 
+async fn load_housekeeping_job(context: &Context) -> Option<Job> {
+    let last_time_str = context
+        .sql
+        .get_raw_config(context, LAST_HOUSEKEEPING)
```

Comment:
Shouldn't `Config::LastHousekeeping` be added to `config.rs` instead of using `get_raw_config`?

Reply:
Storing configs that are only internally used is done sometimes with `get_raw_config`, sometimes by adding a `Config` variant. But I can do it the other way if you say it's better.